src/qtiCommonRenderer/renderers/interactions/ExtendedTextInteraction.js

```diff
@@ -925,15 +926,39 @@ function getState(interaction) {
     return state;
 }
 
-function getCustomData(interaction, data) {
```

Comment:
To avoid a "bigger" breaking change here, maybe we should deprecate this method and delegate to `getData` instead?

Reply:
I don't think it will be a breaking change. The method is named as such internally and exported as `getData`. The internal name does add any difference.